plugins/plugin-site/src/components/PluginHealthScore.jsx

```diff
@@ -1,63 +1,101 @@
-import React, {useState} from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
-import {Progress, Tooltip} from 'reactstrap';
 
-function PluginHealthScore({healthScore, name}) {
-    const score = healthScore.value || 0;
-    const color =
-    score > 80 ? 'success' : score > 60 ? 'warning' : 'danger';
+import './PluginHealthScore.css';
 
-    const tooltipId = `tooltip-${name}`;
-    const [tooltipOpen, setTooltipOpen] = useState(false);
-    const toggle = () => setTooltipOpen(!tooltipOpen);
+function ScoreValue({value}) {
     return (
-        <>
+        <div>
+            {`${value}/100`}
+        </div>
+    );
+}
+
+ScoreValue.propTypes = {
+    value: PropTypes.number.isRequired,
+};
+
+function ScoreComponent({component: {value, reasons}}) {
+    return (
+        <div className="pluginHealth--score-component">
+            <div className="pluginHealth--score-component--reasons">
+                {reasons.map((reason, idx) => {
+                    return (
+                        <span key={idx}>{reason}</span>
+                    );
+                })}
+            </div>
+            <ScoreValue value={value} />
+        </div>
+    );
+}
+
+ScoreComponent.propTypes = {
+    component: PropTypes.shape({
+        value: PropTypes.number.isRequired,
+        weight: PropTypes.number.isRequired,
+        reasons: PropTypes.arrayOf(PropTypes.string),
+    }).isRequired
+};
+
+function ScoreDetail({data: {name, components, value}}) {
+    return (
+        <div className="pluginHealth--score-section">
+            <div className="pluginHealth--score-section--header">
+                <div className="pluginHealth--score-section--header-title">{name}</div>
```

Comment:
Would be nice to make these look more user-friendly, e.g.
```
                <div className="pluginHealth--score-section--header-title">{name.split("-").map(ucFirst).join(" ")}</div>
```
with `ucFirst` imported from [utils](https://github.com/jenkins-infra/plugin-site/blob/8f81ee3d8aecd601f4bfbab6fd2050f39db10b3d/plugins/plugin-site/src/utils/ucfirst.js#L1) would help.

Reply:
Fixed in https://github.com/jenkins-infra/plugin-site/pull/1431/commits/4c90a5e2e0381b0af4f5733a2c058dd0dc0943f7